src/hooks/bridge/useListeningAppMessage.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import { LISTENLING_WEBVIEW_MESSAGE_KEY } from './type';
+import useListeningAppMessage from './useListeningAppMessage';
+
+describe('useListeningAppMessage', () => {
+  const targetType: LISTENLING_WEBVIEW_MESSAGE_KEY = 'foo';
+  const AFTER_MESSAGE_TEXT = 'correct';
+
+  const App = () => {
+    const [state, setState] = useState<boolean>(false);
+    useListeningAppMessage({ targetType, handler: () => setState(true) });
+
+    return <div>{state && AFTER_MESSAGE_TEXT}</div>;
+  };
+
+  beforeEach(() => {
+    Object.defineProperty(window, 'ReactNativeWebView', {
+      value: { postMessage: undefined },
+      writable: true,
+    });
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it('마운트 시 addEventListener가 실행되어야 한다', () => {
+    jest.spyOn(document, 'addEventListener').mockImplementation();
+    jest.spyOn(window, 'addEventListener').mockImplementation();
+    render(<App />);
+
+    expect(document.addEventListener).toBeCalled();
+    expect(window.addEventListener).toBeCalled();
```

Comment:
```suggestion
    const documentAddEventListener = jest.spyOn(document, 'addEventListener');
    const windowAddEventListener = jest.spyOn(window, 'addEventListener');
    render(<App />);

    expect(documentAddEventListener).toBeCalled();
    expect(windowAddEventListener).toBeCalled();
```

저는 이렇게 보통 짜는편이에요.

`jest.spyOn(document, 'addEventListener').mockImplementation()` 요게

`document.addEventListener = jest.fn()`
이거랑 같은거 아닌가요?? spyOn 쓰는이유가 내부 구현은 유지되는거라서 구현은 유지하되 호출되는지만 체크하려구요

Reply:
맞아요 ! 호출 확인을 위해 작성했었어요!

말씀하신 방법이 조금 더 의미에 적합한 방법인 거 같아요 !! 수정해 보겠습니다 ~~! 👍 👍